avocado/core/nrunner.py

```diff
@@ -672,12 +674,15 @@ def __init__(self, identifier, runnable, status_uris=None,
         if known_runners is None:
             known_runners = {}
         self.known_runners = known_runners
+        self.prereqs = []
```

Comment:
One suggestion is to rename this to `requirements` to be more explicity.

Reply:
This is a list of pre-requirements that needs to be fulfilled before this task starts. Do you think calling it `requirements` will not confuse with the test requirements? Whatever is clearer is fine for me

Reply:
Oh, so I got this even more confused here, I thought with `prereqs` you mean `prerequisites` which is pretty much the same as requirements. I would have never expected it means "pre-requirements" and I am not even sure that this means just from the wording. I suggested some other possible names in other comments here but it seems you run into recursion problem where task requirements have their own requirements where multiple names will only exacerbate the situation.

Reply:
Yes, naming has become harder and harder when `requirements` is used in multiple places to mean different things. I'll try my best to make it clearer in the next version.